bindings/dart/tests/opendal_test.dart

```diff
@@ -5,16 +5,15 @@ void main() {
   group('opendal unit test', () {
     group('opendal fs schema', () {
       test('File and Directory functions in fs schema', () async {
-        await RustLib.init();
-        final File = FileManager.initOp(schemeStr: "fs", map: {"root": "/tmp"});
+        final File = await FileManager.init(schemeStr: "fs", map: {"root": "/tmp"});
         var testFile = File("test_1.txt");
         expect(await testFile.exists(), false);
 
         var anotherFile = File("test.txt");
         expect(await anotherFile.exists(), false);
 
-        final Directory =
-            DirectoryManager.initOp(schemeStr: "fs", map: {"root": "/tmp"});
+        final op = Operator(schemeStr: "fs", map: {"root": "/tmp"});
```

Comment:
Hi, if we ultimately need to expose elements like `Operator`, perhaps we should reconsider our API design to determine whether `FileManager` and `DirectoryManager` are truly beneficial.

I'm thinking of some thinkg like:

```dart
final storage = await Storage(schemeStr: "fs", map: {"root": "/tmp"});

// Users can crerate a file/dir factory if they want.
final File = storage.initFile(); // I don't if it's possible.
final Dir = storage.initDir();  // I don't if it's possible.

// Users can crate File/Directory from storage directly too.
var file = storage.newFile();
var dir = storage.newDir();
```

What do you think?